src/main/java/org/sagebionetworks/bridge/exporter/synapse/ColumnDefinition.java

```diff
@@ -0,0 +1,164 @@
+package org.sagebionetworks.bridge.exporter.synapse;
+
+import org.sagebionetworks.repo.model.table.ColumnType;
+
+public class ColumnDefinition  {
+    private String name;
+    private ColumnType columnType;
+    private Long maximumSize;
+    private TransferMethod transferMethod;
```

Comment:
Can we combine TransferMethod and ColumnType? It looks like these map one-to-one, and I can't think of any case where they'd be different. The simplest thing to do is for TransferMethod to have a getColumnType() method that returns the appropriate ColumnType.